CodeReview/src/CodeReviewManager.h

```diff
@@ -28,19 +28,41 @@
 
 #include "codereview_api.h"
 
+#include "VersionControlUI/src/nodes/DiffFrame.h"
+
 #include "nodes/CommentedNode.h"
 
 namespace CodeReview {
 
+using DiffFrames = QList<VersionControlUI::DiffFrame*>;
+
+using OrderedDiffFrames = QList<VersionControlUI::DiffFrame*>;
```

Comment:
I'm not sure, it helps to quickly see what a method is actually returning (e.g. `OrderedDiffFrames Orderings::alphabeticalOrdering(DiffFrames diffFrames)`) but on the other hand the method name should be clear enough.

I can remove the using definitons for the lists and leave the ones for the functions, what do you think?